src/lua/lua_script.rs

```diff
@@ -10,7 +10,9 @@ use super::{
 use crate::lua::fs::Fs;
 use crate::lua::prompt::Prompt;
 use crate::lua::prompt_mask::PromptMask;
+#[cfg(feature = "spellcheck")]
 use crate::lua::spellcheck;
+#[cfg(feature = "spellcheck")]
 use crate::lua::spellcheck::Spellchecker;
```

Comment:
Group these includes so that we only need one feature flag:
`use crate::lua::spellcheck::*;`
Or the squiggly bracket version. I forget the pattern.